recipes-ni/ni-grpc-device/files/run-ptest

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
```

Comment:
Isn't the ptest useful anymore?

Reply:
The competing recipe versions used two different paths for the `run-ptest` file. The modern version uses the file from [here](https://github.com/ni/meta-nilrt/blob/b4f33d2bbe02c9f20847e1fb36c67a010d9b5084/recipes-ni/ni-grpc-device/files/ptest/run-ptest), which is identical to the version one-level-above, used by the static-binary recipe version.

The GH PR doesn't do a great job of showing that I have dropped one and not the other.